.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+labels are especially useful.
+
+You may find an issue is assigned. Please double-check before starting on this issue because somebody else is likely already working on it.
+
+#### Dev environment
+
+When developing, prefer using **Node** ≥ 8 and **yarn**. Writing code with the latest stable Node versions allows us to use newer developer tools.
+
+After [cloning the repository](https://help.github.com/articles/cloning-a-repository/), run `yarn` to install dependencies.
+
+A summary of the scripts:
+
+- To run the example use `yarn dev:example`;
+- The `yarn dev:start` command will build the lib and watch files in bundle and rebuild on changes;
+- Use `yarn test` to run the test suite (powered by [Jest](https://facebook.github.io/jest/));
+- Run `yarn coveralls` to provide test coverage statistics;
+- `yarn build` will build the lib using [Rollup](https://rollupjs.org/guide/en);
+
+This project uses [Prettier](http://prettier.io/) for code formatting. Consider installing an [editor plugin](https://prettier.io/docs/en/editors.html) for the best experience, but code will also be formatted with a precommit script (using [lint-staged](https://github.com/okonet/lint-staged)).
+
+## Commiting
+
+- Never use the `-m <msg>` / `--message=<msg>` flag to `git commit`;
+- Use the present tense ("Add feature" not "Added feature");
+- Use the imperative mood ("Move cursor to..." not "Moves cursor to...");
+- The first line should:
+  - Be capitalized;
+  - Have 50 characters or less;
+  - Omit any trailing punctuation;
+  - Be followed by a blank line.
+- The following lines should have 72 characters or less;
+- Reference issues and pull requests liberally after the first line;
+- When creating a commit try to answer the following questions:
+  - Why is this change necessary?
+  - How does it address the issue?
+  - What side effects (if any) does this change may have?
+
+Example:
+
+```
+Commit message style guide for Git
+
+The first line of a commit message serves as a summary.  When displayed
+on the web, it's often styled as a heading, and in emails, it's
+typically used as the subject. As such, you should capitalize it and
+omit any trailing punctuation. Aim for about 50 characters, give or
+take, otherwise it may be painfully truncated in some contexts. Write
+it, along with the rest of your message, in the present tense and imperative mood: "Fix
+bug" and not "Fixed bug" or "Fixes bug". Consistent wording makes it
+easier to mentally process a list of commits.
+
+Oftentimes a subject by itself is sufficient.  When it's not, add a
+blank line (this is important) followed by one or more paragraphs hard
+wrapped to 72 characters.  Git is strongly opinionated that the author
+is responsible for line breaks; if you omit them, command line tooling
+will show it as one extremely long unwrapped line. Fortunately, most
+text editors are capable of automating this.
+
+When relevant, issues and pull request can be referenced using theirs numbers: #3 #12
```

Comment:
Here we are referencing pull requests and issues in the last line, but the guide is saying to use it literally after the first line.

Reply:
`liberally` not `literally`. It means that it shouldn't matter where this kind of reference is made.

It doesn't make sense to specify a place to make this reference because it varies to each issue.

Reply:
Holy crap, forget about that, missing my glasses.